lib/commits.js

```diff
@@ -1,4 +1,6 @@
 const log = require('./log')
+const paginate = require('./pagination')
+const _ = require('lodash')
```

Comment:
Would it not be better only to require what you need?

Reply:
Performance wise it shouldn't really matter I think? I like the convenience of having all of Lodash under a single char 😅